swap/swap.go

```diff
@@ -90,11 +99,39 @@ type ioData struct {
 
 // Meta returns plugin meta data
 func Meta() *plugin.PluginMeta {
-	return plugin.NewPluginMeta(name, version, pluginType, []string{}, []string{plugin.SnapGOBContentType})
+	return plugin.NewPluginMeta(PluginName, version, pluginType, []string{}, []string{plugin.SnapGOBContentType})
```

Comment:
The option plugin.RoutingStrategy(plugin.StickyRouting) should be added to ensure that there is a 1:1 relationship between running tasks and running plugin instances.